package.json

```diff
@@ -33,7 +33,6 @@
     "readdirp": "~1.0.1"
   },
   "devDependencies": {
-    "tap": "~0.3.3",
-    "cardinal": "~0.3.1"
```

Comment:
Let's keep changes to the feature you're adding. Makes it easier to review.
I can update deps later (although this seems not critical).